adapters/telegram/src/utils.ts

```diff
@@ -7,7 +7,7 @@ const logger = new Logger('telegram')
 export const adaptUser = (data: Telegram.User): User => ({
   userId: data.id.toString(),
   username: data.username,
-  nickname: data.first_name + (data.last_name || ''),
+  nickname: data.last_name ? data.first_name + ' ' + data.last_name : data.first_name,
```

Comment:
Maybe this is better?

```suggestion
  nickname: data.first_name + (data.last_name ? ' ' + data.last_name : ''),
```

Reply:
emmm, I think it might be slower

Reply:
Do you have concrete evidence that the above writing has significant performance issues?

Reply:
idk, anyway it's up to you to decide

Reply:
I think the suggested writing is clearer, easier to maintain and more similar to old version (based on the principle of minimal changes).

Reply:
ok